deploy/kicbase/Dockerfile

```diff
@@ -164,7 +164,7 @@ RUN sh -c "echo 'deb https://download.opensuse.org/repositories/devel:/kubic:/li
     apt-key add - < Release.key && \
     clean-install containers-common catatonit conmon containernetworking-plugins cri-tools podman-plugins crun
 
-# install cri-o based on https://github.com/cri-o/cri-o/blob/release-1.22/README.md#installing-cri-o
+# install cri-o based on https://github.com/cri-o/cri-o/blob/release-1.22.1/README.md#installing-cri-o
```

Comment:
They only update the documentation on minor releases, so this will stay at `1.22`.

Reply:
Great, will keep this at 1.22.